committeeoversight/settings.py

```diff
@@ -157,6 +157,8 @@ def before_send(event, hint):
 
 WAGTAIL_MODERATION_ENABLED = False
 
+DEFAULT_CONGRESS_LENGTH = 668
```

Comment:
Rather than setting this here, I might suggest setting this on congress save using the [timedelta](https://docs.python.org/3/library/datetime.html#timedelta-objectse) between the start and end dates if a length is not passed.

Reply:
Hm, I see you need this value for calculating the handicap. I'll leave this comment here but defer to your judgment on these calculations. Is consistency more important than precision? 

Is there a different approach that _doesn't_ make you choose? Perhaps having the admin input number of days to remove from a session, instead of setting the length, and then handicap could be:

```python
length = end_date - start_date
handicap = (length - inactive_days) / length
```

Reply:
Oh `inactive_days` is a good way of thinking about it! All Congresses are two years, give or take a day, but the `length` is calculated at 20 months, because they generally have 2 inactive months a year. So the approach you're suggesting makes a lot of sense, I think, and helps me get unstuck from thinking about it the way I was previously. I'll try out implementing this.